cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -266,8 +284,16 @@ public synchronized void updateProperties(Id.Namespace namespaceId, NamespaceMet
     }
 
     NamespaceConfig config = namespaceMeta.getConfig();
-    if (config != null && !Strings.isNullOrEmpty(config.getSchedulerQueueName())) {
-      builder.setSchedulerQueueName(config.getSchedulerQueueName());
+    if (config != null) {
+      if (!Strings.isNullOrEmpty(config.getSchedulerQueueName())) {
```

Comment:
This looks duplicate to the `NamespaceHttpHandler`. What need the same checks in two places?